Driverless AI/build-your-own-custom-recipe-tutorial/build-your-own-custom-recipe-tutorial.md

```diff
@@ -208,173 +214,211 @@ This transformer recipe will add three or more numeric columns and give the sum.
-
-- The **CustomTimeSeriesTransformer** function extends the **CustomTransformer** and allows for updating of historical values. This function is very specific to Time Series, such as Facebook Prophet or ARIMA.
+Let's start by creating a transformer recipe class in Python called **SumTransformer** that inherits from the **CustomTransformer** Base Class. This transformer will be responsible for computing the **Summation of multiple Columns**.
 
+1\. Open your text editor and create a new file.
 
-For our custom Summation of Multiple Columns Transformer, we will use the ```CustomTransformer``` along with the following three packages:
-- Numpy
-- Datatable
-- CustomTransformer Class - which is part of Driverless AI
+2\. Save the new file as **sum.py**
 
+3\. Copy and paste the following code into your .py file.
 
 The python code for **Extending the Transformer Base Class** is as follows:
 
 ~~~python
-"""Adds together 3 or more numeric features"""	
-#Extend base class				
+"""Adds together 3 or more numeric features"""				
 from h2oaicore.transformer_utils import CustomTransformer 
 import datatable as dt
 import numpy as np
 
 class SumTransformer(CustomTransformer):
 ~~~
 
-1\. Open your text editor and create a new file.
+The `SumTransformer` class inherits from the `CustomTransformer` Base Class. Prior to creating the `SumTransformer` class, we import the necessary modules:
 
-2\. Save  the new file as **sum_transformer.py**
+- CustomTransformer Class - which is part of Driverless AI transformer_utils
+- numpy
+- datatable
 
-3\. Copy and paste the code above into your .py file. Your text editor should look similar to the page below:
+There are two types of Base Classes for a transformer the genetic custom transformer and the more specialized custom TimeSeries Transformer.
 
-![transformer-extending-base-class](assets/transformer-extending-base-class.jpg)
+- In the generic **CustomTransformer** function, the base class is used for most data transformations, including augmentation, like special holidays. The **CustomTransformer** function is the transformer that will be used the majority of the time. 
+- The **CustomTimeSeriesTransformer** function extends the **CustomTransformer** and allows for updating of historical values. This function is very specific to Time Series, such as Facebook Prophet or ARIMA.
 
-**Note:** 
-- The name of the file and the extension .py should be `sum_transformer.py`
+### Specifying Whether our Recipe is Enabled
+
+We will let Driverless AI know our recipe is enabled.
+
+4\. Copy and paste the following code below the `Extending the Transformer Base Class` section of code in your .py file.
+
+The python code for **Specifying Whether our Recipe is Enabled** is as follows:
+
+~~~python
+@staticmethod
+def is_enabled():
+    return True
+~~~
+
+The `is_enabled` method returns that our recipe is enabled. If this method returns False, our recipe is disabled and the recipe will be completely ignored.
```

Comment:
@james94 rewrite the first sentence to: "The `is_enabled` method returns true when our recipe is enabled."

Reply:
@5675sp made the change. It will be in the new commit.